src/ontology/mondo-edit.obo

```diff
@@ -517559,9 +517562,8 @@ xref: ICD10CM:G37.3 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:
 xref: MEDGEN:1842662 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:592873 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680298 {source="MONDO:equivalentTo", source="MEDGEN:1842662", source="MONDO:MEDGEN"}
-is_a: MONDO:0015342 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! acute transverse myelitis
```

Comment:
This line is duplicated with line 517564. It was removed as part of the normalization. It is ok.